src/components/DetailsCard/CardDesktop.tsx

```diff
@@ -0,0 +1,204 @@
+import React from 'react';
+
+import { Box, Card, Typography, CardContent, CardMedia, makeStyles } from '@material-ui/core/';
+import { Download } from 'react-feather';
+
+import { DetailsItem } from './DetailsItem';
+import { Button } from '../Button';
+
+import { renderSocialMediaIcon } from './renderSocialMediaIcon';
+
+import { FC } from '../../typings/components';
+import { CardProps } from './cardProps';
+
+const useStyles = makeStyles((theme) => ({
+  wrapper: {
+    position: 'relative',
+    height: '100%',
+    width: '100%',
+    paddingTop: theme.spacing(14),
+    background: 'transparent',
+  },
+  card: {
+    background: theme.palette.background.paper,
+    borderRadius: 16,
+    height: '100%',
+    width: '100%',
+    '& .MuiCardContent-root': {
+      height: '100%',
+    },
+    '& .MuiCardContent-root:last-child': {
+      padding: 0,
+    },
+    paddingTop: theme.spacing(11),
+  },
+  avatar: {
+    position: 'absolute',
+    top: 0,
+    left: '50%',
+    marginLeft: theme.spacing(-12.5),
+    width: 200,
+    height: 200,
+    borderRadius: 16,
+    boxShadow: '0px 10px 40px -10px rgba(0,0,0,0.2)',
+  },
+  content: {
+    display: 'flex',
+    flexDirection: 'column',
+    alignItems: 'center',
+    padding: 0,
+  },
+  basicInfo: {
+    background: 'transparent',
+    height: '100%',
+  },
+  moreInfo: {
+    display: 'flex',
+    flexDirection: 'column',
+    justifyContent: 'center',
+    alignItems: 'flex-start',
+    background: theme.palette.background.light,
+    padding: theme.spacing(3),
+    minHeight: 268,
+    width: '100%',
+  },
+  detailsItem: {
+    marginBottom: theme.spacing(2),
+  },
+  detailsItemAvailable: {
+    color: theme.palette.success.main,
+  },
+  name: {
+    paddingTop: theme.spacing(2),
+    textAlign: 'center',
+    wordWrap: 'break-word',
+  },
+  position: {
+    color: theme.palette.primary.main,
+    marginTop: theme.spacing(0.75),
+    textAlign: 'center',
+  },
+  link: {
+    width: 21,
+    height: 21,
+    margin: theme.spacing(0, 1),
+  },
+  iconsWrapper: {
+    display: 'flex',
+    width: '100%',
+    justifyContent: 'center',
+    flexDirection: 'row',
+    alignItems: 'center',
+    marginTop: theme.spacing(2.5),
+  },
+  icon: {
+    boxSizing: 'border-box',
+    background: theme.palette.text.secondary,
+    fill: theme.palette.text.secondary,
+    stroke: theme.palette.text.white,
+    width: 21,
+    height: 21,
+    borderRadius: 3,
+    padding: 3,
+  },
+  buttonWrapper: {
+    display: 'flex',
+    width: '100%',
+    justifyContent: 'center',
+    paddingTop: theme.spacing(1.5),
+  },
+  button: {
```

Comment:
The button component should be reusable, and shouldn't need additional styling. If it's not, catch the author of the component and fix it together 😉

Reply:
Alirghty, I will get going with the author